docs/user/dashboard/lens.asciidoc

```diff
@@ -319,7 +319,12 @@ You can also change the index pattern for a single *Layer*.
 [[why-my-field-x-is-missing-from-the-fields-list]]
 ===== Why is my field X missing from the fields list?
```

Comment:
Isn't this just `Why is my field missing from the list of fields?`

When I read `field X`, I think `x-axis`.

Reply:
I see `X` as generic term, but I see what you mean. Will remove it.